docs/app/Component/ComponentProps.js

```diff
@@ -29,42 +43,28 @@ export default class ComponentProps extends Component {
       }
       type = type && `{${type}}`;
 
-      let required = propConfig.required && (
-          <span className='ui mini red circular label'>required</span>
-        );
-
-      // default value
-      if (propConfig.defaultValue) {
-        defaultValue = (
-          <span>{propConfig.defaultValue.value}</span>
-        );
-        defaultIsComputed = propConfig.defaultValue.computed && (
-            <span className='ui mini gray circular label'>computed</span>
-          );
-      }
+      let required = propConfig.required;
+      let defaultValue = propConfig.defaultValue;
 
-      return (
-        <div key={propName}>
-          <div style={{display: 'flex'}}>
-            <div style={{flex: '4'}}>{name} {required}</div>
-            <div style={{flex: '2'}}>{type}</div>
-            <div style={{flex: '4'}}>{defaultValue} {defaultIsComputed}</div>
-            <div style={{flex: '8'}}>{description}</div>
-          </div>
-        </div>
-      );
+      return {
+        name,
+        type,
+        value,
+        required,
+        defaultValue,
+        description,
+      };
```

Comment:
We now return an array of objects describing each property.  The table will use this array of PropType objects creating a row for every object.